generators/python_functions/spacy_lemmatizer/code_snippet.md

```diff
@@ -0,0 +1,6 @@
+```python
+YOUR_ATTRIBUTE: str = "text"
+
+def spacy_lemmatizer(record):
+    return lemmatized_text = " ".join([token.lemma_ for token in record[YOUR_ATTRIBUTE].text])
```

Comment:
Two changes:

- return statement don't work while assigning a value to the variable you want to return, maybe just `return lemmatized_text` in a new line? :)
- This will not work in general due to the ".text" at the end of line 5. It should not be there since `token.lemma_` works only on the `doc` object.

Reply:
Yes, absolutely right! Changed this. :)